doc/modules/cross_validation.rst

```diff
@@ -263,6 +263,29 @@ Thus, one can create the training/test sets using numpy indexing::
   >>> X_train, X_test, y_train, y_test = X[train], X[test], y[train], y[test]
 
 
+Repeated K-Fold
+---------------
+
+:class:`RepeatedKFold` repeats K-Fold n times. It can be used when one
+requires to run :class:`KFold` n times, producing different splits in
+each repetition.
+
+Example of 2-fold K-Fold repeated 2 times::
+
+  >>> import numpy as np
+  >>> from sklearn.model_selection import RepeatedKFold, KFold
+  >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
+  >>> random_states = [12883823, 28827347]
+  >>> rkf = RepeatedKFold(KFold(n_splits=2), n_repeats=2, random_states=random_states)
```

Comment:
Yes, it would be better to accept `n_splits` to the constructor of `RepeatedKFold` and create an instance of `KFold` inside. Likewise for `RepeatedStratifiedKFold`.

I'll make the changes.